src/App.js

```diff
@@ -5,6 +5,7 @@ import ProductDetails from "./pages/ProductDetails";
 
 import MainMenu from "./components/MainMenu";
 import { Header } from "./components/Layout";
+import { Footer } from "./components/Layout";
```

Comment:
To tak by się tylko przyczepić i potestować te suggestion ;)

```suggestion
import { Header, Footer } from "./components/Layout";
```

Reply:
słusznie, poprawię :)

Reply:
chyba można skorzystać z tego suggestion i scommitować, dlatego skomentowałem by to sprawdzić jak działa :) 
![image](https://user-images.githubusercontent.com/7274216/116355879-c5f89f80-a7fa-11eb-8311-48a6a535be57.png)